src/Components/ColorPicker/ColorPreview.tsx

```diff
@@ -0,0 +1,27 @@
+// Libraries
+import React, {FunctionComponent} from 'react'
+import classnames from 'classnames'
+
+// Types
+import {StandardFunctionProps} from '../../Types'
+
+// Styles
+import './ColorPreview.scss'
+
+interface ColorPreviewProps extends StandardFunctionProps {
+  /** Hex color in #000000 format */
+  color: string
```

Comment:
style suggestion: this comment might become unnecessary and the code might be more self-documenting if the prop was called `hexColor` or `colorHex`, but I don't know all the details around this prop needing to be named `color`

Reply:
I looked more into the code and found that the prop is not limited to hex color., so I updated the comment